pkgs/games/classicube/default.nix

```
@@ -0,0 +1,93 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+  # native build dependencies
+, dos2unix
+, makeWrapper
+  # build dependencies
+, SDL2
+, libGL
+, curl
+, openal
+  # runtime dependencies
```

```suggestion
, dos2unix
, makeWrapper
, SDL2
, libGL
, curl
, openal
```
thats already documented below and the comments can become out of date